assets/src/domain/shared/entities/tickets/helpers/status.ts

```diff
@@ -0,0 +1,28 @@
+import { Ticket } from '@edtrServices/apollo/types';
+import isExpired from '../predicates/isExpired';
+import isOnSale from '../predicates/isOnSale';
+import isPending from '../predicates/isPending';
+import isSoldOut from '../predicates/isSoldOut';
+import isTrashed from '../../../services/predicates/isTrashed';
+import { TICKET_STATUS_ID } from '../constants';
+
+const status = (ticket: Ticket): string => {
+	if (isExpired({ ticket })) {
+		return TICKET_STATUS_ID.EXPIRED;
+	}
+	if (isOnSale(ticket)) {
+		return TICKET_STATUS_ID.ONSALE;
+	}
+	if (isPending(ticket)) {
+		return TICKET_STATUS_ID.PENDING;
+	}
+	if (isSoldOut(ticket)) {
+		return TICKET_STATUS_ID.SOLD_OUT;
+	}
+	if (isTrashed(ticket)) {
```

Comment:
That order of priority is applicable here instead.

Reply:
Updated: https://github.com/eventespresso/event-espresso-core/pull/2419/commits/125695ccaa1f88bc97f31ff983691e7de9ac2583